protocol/v2/ssv/runner/aggregator.go

```diff
@@ -98,16 +101,32 @@ func (r *AggregatorRunner) ProcessPreConsensus(logger *zap.Logger, signedMsg *sp
 		return errors.Wrap(err, "got pre-consensus quorum but it has invalid signatures")
 	}
 
+	// signer must be same for all messages, at least 1 message must be present (this is validated
+	// prior to ProcessPreConsensus call)
+	signer := signedMsg.Messages[0].Signer
 	duty := r.GetState().StartingDuty.(*spectypes.ValidatorDuty)
 
 	logger.Debug("🧩 got partial signature quorum",
-		zap.Any("signer", signedMsg.Messages[0].Signer), // TODO: always 1?
+		zap.Any("signer", signer),
 		fields.Slot(duty.Slot),
 	)
```

Comment:
Answering TODO: looks like signer should be same for all messages (this is validated earlier with `PartialSignatureMessages.Validate`).